lms/envs/common.py

```diff
@@ -4996,6 +4996,8 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'COMPLETE',
 ]
 
+EXTRA_SERVICES_TO_RETIRE_FROM = {}
```

Comment:
Should the default here be an empty list? It looks like the setting in `test.py` is a list of dicts.